tsconfig.json

```diff
@@ -1,5 +1,10 @@
 {
+	"compileOnSave": false,
 	"compilerOptions": {
+		"noFallthroughCasesInSwitch": true,
+		"noImplicitAny": true,
+		"noUnusedLocals": true,
+		"outDir": "./dist/",
 		"target": "es5",
 		"lib": ["dom", "dom.iterable", "esnext"],
 		"allowJs": true,
```

Comment:
Additionally use this - 
```
  "exclude": ["node_modules"]
```

Reply:
I will add it, but is it really necessary when using: `"include": ["src"]`?

Reply:
Yes. No harm in explicitly ignoring node_modules I think